amq-broker-7-image-streams.yaml

```diff
@@ -26,4 +26,4 @@ items:
         version: '1.0'
       from:
         kind: DockerImage
-        name: amq-broker-7/amq-broker-71-openshift:dev
\ No newline at end of file
+        name: amq-broker-7-tech-preview/amq-broker-71-openshift:1.0
```

Comment:
@vbusch imho this should point to RH registry
```
registry.access.redhat.com/amq-broker-7-tech-preview/amq-broker-71-openshift:1.0
```

Reply:
@dsimansk Taking a look at the ose-v1.4.13 tag of jboss-openshift/application-templates I don't see that we have been specific in the past. While I personally prefer to be specific I think being consistent with the overall set of templates that an end developer is going to use is more important.

Reply:
@RoddieKieley we're in the ImageStream definition of `from`:
https://github.com/jboss-openshift/application-templates/blob/ose-v1.4.13/amq/amq63-image-stream.json#L74-L85